screens/signIn/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { connect } from 'react-redux';
+import { login } from '../../actions/session';
+import styles from './styles';
+import Background from '../common/background';
+
+import Input from '../../components/common/input';
+import Button from '../../components/common/button';
+
+
+class SignIn extends React.Component {
+  constructor() {
```

Comment:
You can delete this constructor and pass the state like this.
```suggestion
  state = {
      username: "",
      password: "",
    }
```